src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -958,7 +868,9 @@ private long fetchLatestCommittedOffsetFromSnowflake() {
     try {
       offsetToken = this.channel.getLatestCommittedOffsetToken();
       if (offsetToken == null) {
-        LOGGER.info("OffsetToken not present for channelName:{}", this.getChannelName());
+        LOGGER.warn(
```

Comment:
Is this a warn? This would create confusions IMO.

Reply:
I want to keep this as warn since I would imagine most of the corner cases would come from the no offset token situation, keep this as warn makes a bit easier to search